data/planners/254c7035-de7d-4d76-a888-2c09ba594eca.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+id: 254c7035-de7d-4d76-a888-2c09ba594eca.yml
+name: look ahead
+description: |
+  The look ahead planner decides what abilities to use based on anticipated
+  future rewards.  It takes as input a table of ability rewards, a depth
+  parameter, and a discount factor.  The depth parameter effectively controls
+  the "look-ahead" for the planner as it's scoring each action, and the
+  discount factor controls how the planner weighs future rewards.
+module: plugins.research.app.look_ahead
```

Comment:
We can change `research` to `stockpile` in this line,

Reply:
Nice catch. I just fixed this in the latest commit.